fhir-install/Dockerfile

```
@@ -9,30 +9,29 @@ FROM openliberty/open-liberty:22.0.0.7-kernel-slim-java11-openj9-ubi as base
 
 USER root
 RUN yum install -y unzip
-RUN install -d -o 1001 /opt/ibm-fhir-server
+RUN install -d -o 1001 /opt/fhir-server
 USER 1001
 
 COPY target/fhir-server-distribution.zip /tmp/
 RUN unzip -qq /tmp/fhir-server-distribution.zip -d /tmp && \
-    /tmp/fhir-server-dist/install-fhir.sh /opt/ol/wlp && \
-    mv /tmp/fhir-server-dist/tools /opt/ibm-fhir-server/tools
+    /tmp/fhir-server-dist/install.sh /opt/ol && \
+    mv /tmp/fhir-server-dist/tools /opt/fhir-server/tools
 COPY src/main/docker/ibm-fhir-server/bootstrap.properties /opt/ol/wlp/usr/servers/defaultServer/
-COPY src/main/docker/ibm-fhir-server/bootstrap.sh /opt/ibm-fhir-server/
+COPY src/main/docker/ibm-fhir-server/bootstrap.sh /opt/fhir-server/
 
 # ----------------------------------------------------------------------------
 # Stage: Runnable
 
 FROM openliberty/open-liberty:22.0.0.7-kernel-slim-java11-openj9-ubi
 
 ARG VERBOSE=true
-ARG FHIR_VERSION=5.0.0-SNAPSHOT
+ARG FHIR_SERVER_VERSION=5.0.0-SNAPSHOT
 
 # The following labels are required:
-LABEL name='IBM FHIR Server'
-LABEL vendor='IBM'
-LABEL version="$FHIR_VERSION"
-LABEL summary="Image for IBM FHIR Server with OpenJ9 and UBI 8"
-LABEL description="The IBM FHIR Server is a modular Java implementation of version 4 of the HL7 FHIR specification with a focus on performance and configurability."
+LABEL name='LinuxForHealth FHIR Server'
+LABEL version="$FHIR_SERVER_VERSION"
+LABEL summary="LinuxForHealth FHIR Server with OpenJ9 and UBI 8"
+LABEL description="The LinuxForHealth FHIR Server is a modular Java implementation of version 4 of the HL7 FHIR specification with a focus on performance and configurability."
```

should we be more specific about the FHIR versions we support?

---

Yeah, I'm never sure where it might show up in searches. Thought it might be useful to mention both.

---

I still think I prefer just keeping it at "version 4"... if they came out with a 4.3.1 or a 4.4.0 we'd support those too.

If we did add it, I think most folks know 4.3.0 as "R4B" and so maybe it would become something like this:
```suggestion
LABEL description="The LinuxForHealth FHIR Server is a modular Java implementation of HL7 FHIR versions R4 and R4B with a focus on performance and configurability."
```

---

spoke with robin and he felt stronger than me on this one...merging the suggestion now